docs/content/en/docs/implementing/slo/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Analisis
+description: Understand Analysis in Keptn and how to use them
+weight: 150
+cascade:
+  type: docs
+  currentversion: main
+---
+
+The Keptn Metrics Operator implements an SLO/SLI feature set inspired by Keptn v1 under the name of Analysis.
+With an Analysis Definition you can specify multiple Service Level Objectives that will be evaluated in your Analysis.  
+At the end of the Analysis the status will return whether your objective failed or passed.
```

Comment:
I'd like to do a little better description here -- maybe pull in some text from "Quality Gates" like "allows you to validate a deployment or release using data provided by your observability platform and using quality criteria that you define as Service Level Objectives (SLOs)". But maybe we should just get this much merged and then I can go in and finesse the description a bit?

Reply:
Yes, also the analysis described here works without the need for the rest of the lifecycle controller, but maybe we could as well have some reflection on how to integrate it with evaluations... for sure let's do it after

Reply:
Agreed!